exercises/error-handling/error_handling_test.py

```diff
@@ -6,18 +6,26 @@
 class FileLike(object):
     def __init__(self):
         self.is_open = False
+        self.was_open = False
+        self.did_something = False
 
     def open(self):
-        self.is_open = True
-
-    def __enter__(self):
+        self.was_open = False
         self.is_open = True
 
     def close(self):
         self.is_open = False
+        self.was_open = True
+
+    def __enter__(self):
+        self.open()
 
```

Comment:
If you add `return self` here (plus make the change to `__exit__`), then this class will work with a context manager.